### PR TITLE
docs: add FAQ page and minimum versions for azure-identity sdks

### PR DESCRIPTION
**Pull request by chewong**
**Reason for Change**:

Adding FAQ page and minimum versions for azure-identity sdks.

**Requirements**

- [x] squashed commits
- [ ] included documentation
- [ ] added unit tests and e2e tests (if applicable).

**Issue Fixed**:

**Please answer the following questions with yes/no**:

Does this change contain code from or inspired by another project? If so, did you notify the maintainers and provide attribution?

- [ ] yes
- [ ] no

**Notes for Reviewers**:

### CODE REVIEW

File: docs/book/src/faq.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
...
+
+## Why is managed identity not supported by Azure Workload Identity?
+
+This is due to the limitation of Azure AD, rather than the Azure Workload Identity project. Azure AD is currently working on enabling workload identity federation for managed identity and Azure Workload Identity will support the feature as soon as it is available.
+
+## How does the azwi-cli differ from the azure-cli?
+
+The azwi-cli tool is specific to the Azure Workload Identity support in Kubernetes to group several manual steps (e.g. the creation of federated identity credential, annotated service accounts, etc) and automate them. Comparing with the azure-cli, it does not have an official command to add/delete federated identity (configuring federated identity credential with `az rest` is available [here](https://docs.microsoft.com/en-us/azure/active-directory/develop/workload-identity-federation-create-trust))
+
+Azure CLI and AKS are currently working on the above requirements, as well as an Azure CLI extension that natively integrate this project with AKS clusters.
```
Comment (aramase) [inline docs/book/src/faq.md:11]: @miwithro @bcho Could you help review this statement we're publishing to our doc?

File: docs/book/src/topics/language-specific-examples/azure-identity-sdk.md [hunk 2]
```diff
@@ -1,4 +1,11 @@
 # Azure Identity SDK
 
-_Coming soon_
+| Language              | Library                                                               | Minimum Version                                                                                                     |
+| --------------------- | --------------------------------------------------------------------- | ------------------------------------------------------------------------------------------------------------------- |
+| Go                    | N/A                                                                   | N/A                                                                                                                 |
```
Comment (aramase) [inline docs/book/src/topics/language-specific-examples/azure-identity-sdk.md:5]: I see an [azidentity release](https://github.com/Azure/azure-sdk-for-go/releases/tag/sdk%2Fazidentity%2Fv0.13.0) for go. @udayxhegde Could we use this preview url for the minimum version here?

File: docs/book/src/faq.md (hunk 1 above)
Comment (bcho) [reply docs/book/src/faq.md:11]: LGTM, and yep I am working on the change in azure-cli side

**Review by aramase — APPROVED** [review]
We can update the go library minimum version in a follow up PR

**Pull request merged**